src/test/java/bio/terra/integration/DataRepoFixtures.java

```diff
@@ -940,6 +942,62 @@ private DataRepoResponse<DatasetDataModel> retrieveDatasetDataByIdRaw(
     return dataRepoClient.get(user, url + queryParams, new TypeReference<>() {});
   }
 
+  public void assertColumnTextValueCount(
+      TestConfiguration.User user,
+      UUID datasetId,
+      String tableName,
+      String columnName,
+      String columnTextValue,
+      int expectedValueCount)
+      throws Exception {
+    if (expectedValueCount == 0) {
+      assertThat(
+          "Column value is not present",
+          retrieveColumnTextValues(user, datasetId, tableName, columnName),
+          not(hasItem(columnTextValue)));
+      return;
+    }
+    int count =
+        retrieveColumnStats(user, datasetId, tableName, columnName, null).getValues().stream()
+            .filter(val -> val.getValue() != null ? val.getValue().equals(columnTextValue) : false)
+            .findFirst()
+            .orElseThrow()
```

Comment:
Is it worth throwing something more specific to aide in test debugging if we encounter this case?